movie-showcase/src/routes/managing/+server.ts

```diff
@@ -0,0 +1,15 @@
+import { API_KEY } from '$env/static/private'
```

Comment:
To make it easy for users to start up this project, you could add a `.env.dist` file which is a placeholder file for environment variables.

The file should be exactly like the `.env` file, but without the values of course.

Reply:
I've made a .env.example file and put instructions in the readme